src/Dfe.EarlyYearsQualification.Web/Controllers/QualificationDetailsController.cs

```diff
@@ -34,10 +34,16 @@ public async Task<IActionResult> Get()
 
         return View(model);
     }
-    
+
     [HttpPost]
     public IActionResult Refine(string refineSearch)
     {
+        if (!ModelState.IsValid)
+        {
+            logger.LogError("Invalid model state");
+            return RedirectToAction("Index", "Error");
+        }
+
```

Comment:
On the qualification list page, this method is called when they enter something in the search box. If a user enters a value in the text box it refines the qualification list. In order to get back to the original list they can remove the search criteria and press enter. With this bit added in it redirects to the error page which we don't want as the search value is null which causes the model state to be invalid . Please can this be removed? :-)

Reply:
As discussed. Removed redirect, just log. Also, allowed for passing in null `refineSearch` param, which I think is how .NET will treat empty string.